modules/next/src/Plugin/Next/PreviewUrlGenerator/SimpleOauth.php

```diff
@@ -208,9 +207,10 @@ protected function getScopesForCurrentUser() {
    * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
    * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
    */
-  protected function getAdminRole() {
+  protected function getAdminRole(): ?string {
```

Comment:
@apathak18 Does this returns an array or string ? If then, isin't it should be ?
```suggestion
  protected function getAdminRole(): ?array {
```

Reply:
The function description says: `Returns an array of admin roles.` which seems to be incorrect and therefore I got confused, whereas it's returning first role_id i.e string. Looks good. we can move it ahead.